python/mleap/sklearn/logistic.py

```diff
@@ -75,12 +75,23 @@ def serialize_to_bundle(self, transformer, path, model_name):
         inputs = [{
                   "name": transformer.input_features,
                   "port": "features"
-                }]
-
-        outputs = [{
-                  "name": transformer.prediction_column,
-                  "port": "prediction"
-                }]
+                  }]
+
+        outputs = list()
+        outputs.append({
+            "name": transformer.prediction_column,
+            "port": "prediction"
+        })
+
+        outputs.append({
+            "name": "raw_prediction",
+            "port": "raw_prediction"
+        })
+
+        outputs.append({
+            "name": "probability",
+            "port": "probability"
+        })
```

Comment:
Just wanted to make sure, this isn't going to generate the raw predictions/probabilities in sklearn, it's just going to serialize a bundle that at scoring time will generate those. Is that the intention of this change?

Reply:
Hi @ancasarb – that's correct. This just tell's the loaded JVM MLeap bundle to produce predicted and raw probabilities at scoring time.

Reply:
Makes sense, thanks!